cmd/account/mgmt/account-unassign.go

```diff
@@ -0,0 +1,407 @@
+package mgmt
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/iam"
+	"github.com/aws/aws-sdk-go/service/organizations"
+	"github.com/aws/aws-sdk-go/service/resourcegroupstaggingapi"
+	"github.com/openshift/osdctl/pkg/printer"
+	awsprovider "github.com/openshift/osdctl/pkg/provider/aws"
+	"github.com/spf13/cobra"
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+)
+
+func newCmdAccountUnassign(streams genericclioptions.IOStreams, flags *genericclioptions.ConfigFlags) *cobra.Command {
+	ops := newAccountUnassignOptions(streams, flags)
+	accountUnassignCmd := &cobra.Command{
+		Use:               "unassign",
+		Short:             "Unassign account to user",
+		DisableAutoGenTag: true,
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdutil.CheckErr(ops.complete(cmd, args))
+			cmdutil.CheckErr(ops.run())
+		},
+	}
+	ops.printFlags.AddFlags(accountUnassignCmd)
+	accountUnassignCmd.Flags().StringVarP(&ops.payerAccount, "payer-account", "p", "", "Payer account type")
+	accountUnassignCmd.Flags().StringVarP(&ops.username, "username", "u", "", "LDAP username")
+	accountUnassignCmd.Flags().StringVarP(&ops.accountID, "account-id", "i", "", "Account ID")
+
+	return accountUnassignCmd
+}
+
+type accountUnassignOptions struct {
+	awsClient    awsprovider.Client
+	username     string
+	payerAccount string
+	accountID    string
+
+	flags      *genericclioptions.ConfigFlags
+	printFlags *printer.PrintFlags
+	genericclioptions.IOStreams
+}
+
+func newAccountUnassignOptions(streams genericclioptions.IOStreams, flags *genericclioptions.ConfigFlags) *accountUnassignOptions {
+	return &accountUnassignOptions{
+		flags:      flags,
+		printFlags: printer.NewPrintFlags(),
+		IOStreams:  streams,
+	}
+}
+
+func (o *accountUnassignOptions) complete(cmd *cobra.Command, _ []string) error {
+	if o.payerAccount == "" {
+		return cmdutil.UsageErrorf(cmd, "Payer account was not provided")
+	}
+	if o.username == "" && o.accountID == "" {
+		return cmdutil.UsageErrorf(cmd, "Please provide either an username or account ID")
+	}
+	return nil
+}
+
+func (o *accountUnassignOptions) run() error {
```

Comment:
This is not tested. For example what happens when you try to unassign the same account twice? e.g. the first time let's say `deleteSigningCert` threw an error. What would happen the next time? Would that throw error (because it's the SignCert has already been deleted?